pkgs/tools/security/b2sum/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, openmp }:
+
+stdenv.mkDerivation rec {
+  version = "20160619";
```

Comment:
Since this release, upstream has added ARM NEON implementations of BLAKE2b and BLAKE2s which should improve performance on aarch64: https://github.com/BLAKE2/BLAKE2/commit/7965d3e6e1b4193438b8d3a656787587d2579227

Also a bug fix for BLAKE2bp which previously returned incorrect results when `-l` option was set: https://github.com/BLAKE2/BLAKE2/pull/36

To incorporate these changes into Nixpkgs version, could you use version from master?

Reply:
oh my, why didn’t they make a new release then :/
we should probably tell the Homebrew people